rclcpp/src/rclcpp/duration.cpp

```diff
@@ -249,7 +249,17 @@ Duration::to_rmw_time() const
 Duration
 Duration::from_seconds(double seconds)
 {
-  return Duration(static_cast<int64_t>(RCL_S_TO_NS(seconds)));
+  Duration ret;
+  ret.rcl_duration_.nanoseconds = static_cast<int64_t>(RCL_S_TO_NS(seconds));
+  return ret;
+}
+
+Duration
+Duration::from_nanoseconds(rcl_duration_value_t seconds)
```

Comment:
Tiny nitpick: I would have the variable name match the units
```suggestion
Duration::from_nanoseconds(rcl_duration_value_t nanoseconds)
```

Reply:
thanks